backend/app/models/dynamodb.py

```
@@ -0,0 +1,76 @@
+from pydantic import BaseModel
+from typing import Optional, Dict, Any
```

Ruff (0.14.3)

`typing.Dict` is deprecated, use `dict` instead

(UP035)